packages/starlight/components/Icons.ts

```diff
@@ -164,6 +164,11 @@ export const BuiltInIcons = {
+	jira: '<path d="M7.75054 16.2932H5.6174c-3.2175 0-5.5256-1.97068-5.5256-4.85653h11.4713c.59408 0 .9786.42298.9786 1.02091V24c-2.86757 0-4.79115-2.32252-4.79115-5.5602Zm5.66595-5.73612h-2.1341c-3.21654 0-5.52464-1.93511-5.52464-4.82096h11.47032c.59409 0 1.01418.38645 1.01418.98534v11.54242c-2.86758 0-4.82576-2.32252-4.82576-5.5602zm5.70055-5.70056h-2.1341c-3.2175 0-5.5256-1.97068-5.5256-4.85652h11.47128c.5941 0 .97958.42201.97958.98534v11.54242c-2.86758 0-4.79116-2.32252-4.79116-5.5602z"/>',
+	storybook:
+		'<path d="m20.34961.00195-1.32422.08203.0957 2.77149c.00347.09935-.07447.18208-.17382.18555-.04255.00148-.08375-.01273-.1172-.03907l-.89257-.70312-1.05664.80273c-.0792.06008-.19187.04405-.25195-.03515-.0253-.03334-.0369-.07538-.03516-.1172l.11133-2.71874L3.5 1.05469c-.65344.04084-1.15541.59575-1.13086 1.25l.74414 19.81054c.02355.62748.52311 1.13199 1.1504 1.16016l16.10741.72266c.01804.00083.03664.00195.0547.00195.6664 0 1.20507-.54061 1.20507-1.20703V1.20703c0-.0251-.00039-.05111-.00195-.07617-.04157-.66512-.61418-1.17048-1.2793-1.1289zm-7.98633 4.08203c3.14117 0 4.85742 1.68005 4.85742 4.87696-.4245.32976-3.58789.5554-3.58789.08594.06684-1.79137-.73413-1.8711-1.17969-1.8711-.42327 0-1.13671.12823-1.13671 1.0879 0 2.36716 6.10351 2.23863 6.10351 7.01562 0 2.68705-2.18207 4.17187-4.9668 4.17187-2.87383 0-5.3852-1.16231-5.10156-5.19336.1114-.47335 3.76367-.36085 3.76367 0-.04455 1.66342.33503 2.15235 1.29297 2.15235.73517 0 1.06836-.40547 1.06836-1.0879 0-2.43114-6.01367-2.51556-6.01367-6.97265 0-2.5591 1.75923-4.26563 4.9004-4.26563z"/>',
```

Comment:
Ran each of these through [SVGOMG](https://jakearchibald.github.io/svgomg/) and was able to compress them a bit without loss of quality:

```suggestion
	confluence:
		'<path d="M.85 18.07.1 19.32a.76.76 0 0 0-.1.28.76.76 0 0 0 .02.28.75.75 0 0 0 .33.46l4.97 3.07a.76.76 0 0 0 .86-.03.75.75 0 0 0 .2-.23l.73-1.23c1.97-3.23 3.97-2.83 7.54-1.14l4.93 2.34a.76.76 0 0 0 .6.03.76.76 0 0 0 .43-.4l2.36-5.35a.75.75 0 0 0 .02-.57.77.77 0 0 0-.38-.43c-1.04-.49-3.1-1.45-4.96-2.36C10.9 10.8 5.2 11 .85 18.07Z"/><path d="m23.15 5.94.75-1.25a.77.77 0 0 0 .08-.57.76.76 0 0 0-.13-.27.76.76 0 0 0-.22-.2L18.67.6a.76.76 0 0 0-.29-.1.77.77 0 0 0-.57.13.77.77 0 0 0-.2.23l-.73 1.22c-1.98 3.25-3.96 2.86-7.53 1.16L4.42.89a.78.78 0 0 0-.59-.03.76.76 0 0 0-.26.15.76.76 0 0 0-.18.24L1.02 6.61a.77.77 0 0 0-.02.57c.04.1.09.18.15.25a.76.76 0 0 0 .24.18c1.04.5 3.11 1.45 4.96 2.36 6.73 3.26 12.44 3.04 16.8-4.03z"/>',
	jira: '<path d="M7.75 16.3H5.62C2.4 16.3.09 14.31.09 11.43h11.47c.6 0 .98.42.98 1.02V24c-2.87 0-4.79-2.32-4.79-5.56Zm5.67-5.74h-2.14c-3.21 0-5.52-1.94-5.52-4.82h11.47c.6 0 1.01.38 1.01.98v11.54c-2.87 0-4.82-2.32-4.82-5.56zm5.7-5.7h-2.14c-3.21 0-5.52-1.97-5.52-4.86h11.47c.6 0 .98.42.98.99v11.54c-2.87 0-4.8-2.32-4.8-5.56z"/>',
	storybook:
		'<path d="m20.35 0-1.32.08.1 2.78a.18.18 0 0 1-.3.14l-.9-.7-1.05.8a.18.18 0 0 1-.25-.03.18.18 0 0 1-.04-.12l.12-2.72-13.21.82A1.2 1.2 0 0 0 2.37 2.3l.74 19.82a1.2 1.2 0 0 0 1.15 1.16l16.11.72h.06c.66 0 1.2-.54 1.2-1.2V1.12A1.2 1.2 0 0 0 20.35 0zm-7.99 4.08c3.14 0 4.86 1.68 4.86 4.88-.42.33-3.59.56-3.59.09.07-1.8-.73-1.87-1.18-1.87-.42 0-1.13.12-1.13 1.08 0 2.37 6.1 2.24 6.1 7.02 0 2.69-2.18 4.17-4.97 4.17-2.87 0-5.38-1.16-5.1-5.2.11-.47 3.77-.35 3.77 0-.05 1.67.33 2.16 1.29 2.16.73 0 1.07-.4 1.07-1.09 0-2.43-6.02-2.51-6.02-6.97 0-2.56 1.76-4.27 4.9-4.27z"/>',
```